ci/vmtest/run_selftests.sh

```diff
@@ -98,6 +99,17 @@ test_verifier() {
   foldable end test_verifier
 }
 
+run_veristat() {
+  foldable start run_veristat "Running veristat"
+
+  globs=$(awk '/^#/ { next; } { print $0 ".bpf.o"; }' ./veristat.cfg)
+  mkdir -p ${OUTPUT_DIR}
+  ./veristat -o csv -q -e file,prog,verdict,states ${globs} > ${OUTPUT_DIR}/veristat.csv
```

Comment:
Can we please use long options, if available?

Reply:
Done